web-app/django/VIM/apps/instruments/templates/instrument_list.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+
+</head>
+<body>
+    <h1>Instrument List</h1>
+    <ul>
+        {% for instrument in instruments %}
+            <li>{{ instrument.id }}, 
+                {% for instrumentname in instrument.instrumentname_set.all %}
+                    {{ instrumentname.language.en_label }}:{{ instrumentname.name }}
```

Comment:
Some thoughts, all purely cosmetic:
- do we want a space after the colon on this line?
- Do we want to visually distinguish the language and the name?

I gather that currently, we're going to get captions that look like "English:Guitar". We might want "English: **Guitar**". Or even "EN: **Guitar**", where the EN displays "English" on mouseover (via `<span title="English">` or something)

Reply:
All good points...since this is a placeholder template that will never be used (and @yinanazhou has already replaced this in another branch) I won't worry about it here.

Reply:
You are going to want to install Django Debug Toolbar and check how many SQL queries this triggers. Better to do it now and get your `select_related`s in place before wondering why your production site is so slow. :-D